test/runtests.jl

```diff
@@ -424,6 +424,18 @@ end
 @test D(x -> x * D(y -> x * y, 2), 1) === 2
 @test D(x -> x * foo_bar_identity(x), 1) === 2.0
 
+############################################################################################
+
+# issue #51
+@context GemvCtx
+using LinearAlgebra
+
+α, β = 1.0, 2.0
+A, X, Y = rand(Float64, 10, 10), rand(Float64, 10), rand(Float64, 10)
+Y_copy = copy(Y)
+@test overdub(GemvCtx(), LinearAlgebra.BLAS.gemv!, 'T', α, A, X, β, Y) ==
```

Comment:
Since `gemv!` is modifying, you may want separate output arrays and also verify that they are equal.